bigchaindb/backend/mongodb/connection.py

```diff
@@ -143,6 +145,11 @@ def _connect(self):
             raise ConfigurationError from exc
 
 
+MONGO_OPTS = {
+    'socketTimeoutMS': 20000,
+}
```

Comment:
I'm afraid of possible side effects since the secondary replicas are always lagging behind the primary.